tgnnexplainer/xgraph/method/subgraphx_tg.py

```diff
@@ -457,7 +450,7 @@ def explain(self,
                                        c_puct=self.c_puct,
                                        score_func=self.tgnn_reward_wraper,
                                        device=self.device,
-                                       candidate_initial_weights=self.candidate_initial_weights, # BUG: never pass through this parameter?????
```

Comment:
Not a bug, these weights are used to sort the candidate events in `self._select_expand_candidates`.